app/controllers/v0/evss_claims_controller.rb

```diff
@@ -25,7 +28,10 @@ def show
 
     def request_decision
       claim = EVSSClaim.for_user(current_user).find_by(evss_id: params[:id])
-      raise Common::Exceptions::RecordNotFound, params[:id] unless claim
+      unless claim
+        Raven.tags_context(team: 'benefits-memorial-1a') # tag sentry logs with team name
```

Comment:
only thing is, did you mean this to be `1a`? Might be a type :)

Reply:
whoopsies- thanks!